app/services/study_edge_service.rb

```diff
@@ -2,11 +2,10 @@
 LLONG_MAX = 9223372036854775807 # rubocop:disable Style/NumericLiterals
 MAX_PAGE_SIZE = 200
 
-
 class StudyEdgeService
   # @param params - hash representing SearchInputType with symbols as keys.
-  def initialize(params)
-    @params = normalize_params(params).freeze
+  def initialize(params = {})
```

Comment:
this is the meat of the change i put into this file. again, there's a lot more here because of autoformatting.